tools/test/examples/update.py

```diff
@@ -369,8 +362,22 @@ def upgrade_example(github, example, tag, ref,
             return_code = run_cmd(push_cmd)
             
             if not return_code:
-                if user != 'ARMmbed':                                
-                    body = "Please test/merge this PR and then tag Master with " + tag
+                # If the user is not ARMmbed then a fork is being used
+                if user != 'ARMmbed': 
+                    
+                    upstream_repo = 'ARMmbed/'+ example['name']
+                    update_log.debug("Upstream repository: %s", upstream_repo)
+                    # Check access to mbed-os repo
+                    try:
+                        repo = github.get_repo(upstream_repo, False)
+
+                    except:
+                        update_log.error("Upstream repo: %s, does not exist - skipping", upstream_repo)
+                        return False
+           
+                    body = "Please test this PR.\n"
+                    body += "If successful then merge, otherwise provide a known issue.\n"
+                    body += "Once you get notification of the release being made public then tag Master with " + tag
```

Comment:
Do you really think this is going to be the only case where we emit a blob of text? I think there may be more when we extend this script later.